pkg/report/github/github.go

```diff
@@ -105,8 +105,22 @@ func (w Writer) Write(ctx context.Context, report types.Report) error {
 		manifest.Name = string(result.Type)
 		// show path for language-specific packages only
 		if result.Class == types.ClassLangPkg {
-			manifest.File = &File{
-				SrcLocation: result.Target,
+			if report.ArtifactType == ftypes.ArtifactContainerImage {
+				// Replacing `source_location` by the image name and tag
+				image_reference := strings.Join(report.Metadata.RepoTags, ", ")
+				image_with_hash := strings.Join(report.Metadata.RepoDigests, ", ")
+				_, image_hash, found := strings.Cut(image_with_hash, "@")
+				if found {
+					image_reference += "@" + image_hash
```

Comment:
Why are you doing this rather than just using RepoDigests directly? Mind adding a comment?

Reply:
`RepoDigests` looks like `<registry>/<image_name>@sha256:<image_hash>`.
Whereas `RepoTag` looks like looks like `<registry>/<image_name>:<image_tag>`.

So if only using `RepoDigests` it means loosing the `<image_tag>`, I'll add a comment.